src/engine/behavior_script.c

```diff
@@ -971,39 +993,21 @@ void cur_obj_update(void) {
         obj_update_gfx_pos_and_angle(gCurrentObject);
     }
 
-    if (objFlags & OBJ_FLAG_UCODE_LARGE) {
-        gCurrentObject->header.gfx.node.flags &= ~GRAPH_RENDER_UCODE_REJ;
-    } else {
-        gCurrentObject->header.gfx.node.flags |=  GRAPH_RENDER_UCODE_REJ;
+    COND_BIT((!(objFlags & OBJ_FLAG_UCODE_LARGE       )), gCurrentObject->header.gfx.node.flags, GRAPH_RENDER_UCODE_REJ         );
+    COND_BIT((  objFlags & OBJ_FLAG_SILHOUETTE         ), gCurrentObject->header.gfx.node.flags, GRAPH_RENDER_SILHOUETTE        );
+    COND_BIT((  objFlags & OBJ_FLAG_OCCLUDE_SILHOUETTE ), gCurrentObject->header.gfx.node.flags, GRAPH_RENDER_OCCLUDE_SILHOUETTE);
```

Comment:
![image](https://user-images.githubusercontent.com/79979276/135538716-6c0dc629-f50b-4b06-8445-1a1351cf9881.png)
these lines are pretty long, and probably more legible without the spacing. non blocking